syntax = "proto3";

package text2video;

message VideoRequest {
  string prompt = 1;
  string audio_path = 2;
  string filter_option = 3;
}

message VideoResponse {
  string video_path = 1;
  string message = 2;
  int32 status_code = 3;
}

service VideoGenerator {
  rpc Generate (VideoRequest) returns (VideoResponse);
}
